src/DependencyInjection/TailwindExtension.php

```diff
@@ -70,7 +70,13 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->end()
                 ->scalarNode('binary_version')
                     ->info('Tailwind CLI version to download - null means the latest version')
-                    ->defaultValue('v3.4.17')
+                    ->defaultNull()
+                    ->beforeNormalization()
+                        ->ifString()
+                        ->then(static function (string $version): string {
+                            return 'v'.ltrim($version, 'vV');
```

Comment:
Can I use something simple like v4? Like with Composer, I don't want to be bothered with exact versions

Reply:
This is probably also important so we don't need to constantky update the recipe

Reply:
I think it's important this be a "locked" version so all developers are using the same binary (there's a few issues about this).

If we allow composer-style versioning, we'd need a separate thing to "lock" it, which means an update command. I don't know we want to go down this road.

If anything, we could provide an update command that updates this value in config to the latest version:

- `tailwind:update`: updates from say 3.4.1 to latest 3.x
- `tailwind:update --major`: updates from say 3.4.1 to true latest

Reply:
> I think it's important this be a "locked" version so all developers are using the same binary (there's a few issues about this).

You could opt into a lock by specifying the exact version, but everyone else could specify just 4 or 4.1